style.css

```diff
@@ -0,0 +1,1651 @@
+  align-items: flex-start;
+  padding: 0 24px;
+  width: 327px;
+  height: 320.15px;
+  box-sizing: border-box;
+}
+
+.event7 {
+  box-sizing: border-box;
+  width: 327px;
+  height: 72px;
+  padding: 0;
+  margin-left: -25px;
+  display: flex;
+  gap: 80px;
+  list-style-type: none;
+}
+
+.event7 h3 {
+  height: 24px;
+  right: 78px;
+  top: calc(50% - 24px / 2);
+  font-family: "Poppins", sans-serif;
+  font-style: normal;
+  font-weight: bold;
+  font-size: 20px;
+  line-height: 24px;
+  display: flex;
+  align-items: center;
+  color: #000;
+  left: 12px;
+}
+
+.event7 img {
+  width: 24px;
+  height: 24px;
+  top: 8px;
+  left: 8px;
+}
+
+.dropdown {
+  border: none;
+  outline: none;
+  color: white;
+  background-color: #fff;
+  cursor: pointer;
+}
+
+.dropdown:hover {
+  transform: scale(1.1);
+  transition: transform 0.3s ease-in-out;
+}
+
+.language-list {
+  display: flex;
+  flex-direction: column;
+  align-items: flex-start;
+  justify-content: center;
+  padding: 0;
+  gap: 12px;
+  width: 327px;
+  height: 248.15px;
+  top: 72px;
+  margin-top: -40px;
+  margin-left: -24px;
+  box-sizing: border-box;
+}
+
+.JavaScript1 {
+  display: flex;
+  align-items: center;
+  gap: 20px;
+  width: 255px;
+  height: 66.72px;
+  background: #f7f7f9;
+  border-radius: 8px;
+  padding: 0 36px;
+  box-sizing: border-box;
+  margin-top: 50px;
+}
+
+.JavaScript1 img {
+  width: 48px;
+  height: 48px;
+  left: 12px;
+  top: calc(50% - 48px / 2 - 0.36px);
+}
+
+.JavaScript1 li,
+.HTML1 li,
+.CSS1 li {
+  height: 24px;
+  left: 76px;
+  right: 12px;
+  top: calc(50% - 24px / 2 - 0.36px);
+  font-family: "Poppins", sans-serif;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 15px;
+  line-height: 20px;
+  display: flex;
+  align-items: center;
+  color: #253858;
+  list-style-type: none;
+}
+
+.HTML1 {
+  box-sizing: border-box;
+  display: flex;
+  align-items: center;
+  gap: 20px;
+  width: 255px;
+  height: 66.72px;
+  background: #f7f7f9;
+  border-radius: 8px;
+  padding: 0 36px;
+  top: 90.72px;
+  left: 12px;
+}
+
+.HTML1 img {
+  width: 48px;
+  height: 48px;
+  left: 12px;
+  top: calc(50% - 48px / 2 - 0.36px);
+  background: url(html5-512.png);
+}
+
+.CSS1 {
+  box-sizing: border-box;
+  display: flex;
+  align-items: center;
+  gap: 20px;
+  width: 255px;
+  height: 66.72px;
+  background: #f7f7f9;
+  border-radius: 8px;
+  padding: 0 36px;
+  top: 169.43px;
+  left: 12px;
+}
+
+.CSS1 img {
+  width: 48px;
+  height: 48px;
+  top: calc(50% - 48px / 2 - 0.36px);
+  background: url(images.jpg);
+  left: 12px;
+}
+
+.lang {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  margin-left: -20px;
+  gap: 12px;
+  padding: 0;
+}
+
+.lang li {
+  list-style-type: none;
+}
+
+.event18 {
+  box-sizing: border-box;
+  width: 327px;
+  height: 72px;
+  padding: 0 24px;
+  margin-left: -16px;
+  display: flex;
+  align-items: initial;
+  gap: 80px;
+  margin-top: -10px;
+  border-bottom: 1px solid rgb(223, 225, 230);
+}
```

Comment:
- **[OPTIONAL]** Kindly add the right narrows to be similar to the Figma template 👇🏽 

 
![image](https://github.com/jovasal/Finish-Mob-Figma/assets/58642949/cdd9fd9b-41f7-4d54-9461-56742d654efd)

Reply:
I forgot to copy the image for that.

Actually is working fine:

![image](https://github.com/jovasal/Finish-Mob-Figma/assets/35693330/a26e3d07-f3c1-4687-b9cb-bf0583ea17a5)